### PR TITLE
July security update 7B

### PR DESCRIPTION
**Pull request by dansisson**

### CODE REVIEW

**Review by Karl-WE — CHANGES_REQUESTED** [review]
understand the Core OS is the same but in a docs for Azure Stack HCI, even without a GUI, is it necessary to point out Windows 11 removed features?

File: azure-stack/hci/security-update/hci-security-update-jul-2024.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+- **Input Method Editor (IME)**. The candidate list fails to show or shows it in the wrong position.
+
+- **Windows Presentation Foundation (WPF)**. A malformed Human Interface Device (HID) descriptor causes WPF to stop responding.
+
+- **Handwriting panels and touch keyboards**. They do not appear when you use the tablet pen.
+
+- ***HKLM\Software\Microsoft\Windows\DWM ForceDisableModeChangeAnimation (REG_DWORD)***. This is a new registry key. When you set its value to `1` (or a non-zero number), it turns off the display mode change animation. If the value is `0` or the key does not exist, the animation is set to on.
+
+- **Remote Desktop MultiPoint Server**. A race condition causes the service to stop responding.
```
Comment (Karl-WE) [inline azure-stack/hci/security-update/hci-security-update-jul-2024.md:36]: these might be obsolete

**Review by Karl-WE — COMMENTED** [review]
would propose a deletion.

**Comment by Court72** [comment]
@alkohli

Can you review the proposed changes? Note that the new content needs to be moved into the private repo for staging, validation, Acrolinx, and to get it merged.

If you'd like the PR review team's help [moving the commits](https://review.learn.microsoft.com/en-us/help/contribute/contribute-how-to-manage-act-guidelines-public-pr?branch=main#pull-public-commits-to-the-private-repo-for-updates-staging-publication), email the publicprs alias.
	
#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Comment by alkohli** [comment]
@Court72 - could you please close this PR? The content from this PR is captured in another PR and that PR has gone live.  The person who opened this PR has different sets of creds now and is not able to close this PR. Thank you!

**Pull request closed without merging**